src/simulation.py

```diff
@@ -16,6 +16,8 @@ def __init__(self):
         self.population = Population(self.config)
         self.screen = screen_init(self.config)
         self.running = False
+        self.generation = 0
```

Comment:
More descriptive variable name: 
```suggestion
        self.generation_counter = 0
```